decentraland/unity-renderer
# feat: removed color grading as a setting.. left enabled by default

## What does this PR change?

Closes #1699 
Removes color grading setting. Left enabled by default (volume tonemapping enabled in Environment.prefab)

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/remove-color-grading-setting
2. Open settings menu pressing TAB
3. Go to graphics settings
4. Color grading should not be visible
5. Go to unity an press play
6. Search for `t:Volume` in hierarchy window, and look for `PPVolume`
7. Check that tonemapping is enabled in Volume component

If tonemapping is disabled you should check the final rendering a bit brighter.

With tonemapping enabled:
![tonemapping-enabled](https://user-images.githubusercontent.com/56365551/148603375-78fb90ad-af2a-456e-bbc4-e30f752765b9.jpg)

With tonemapping disabled:
![tonemapping-disabled](https://user-images.githubusercontent.com/56365551/148603396-71d1e28e-0a09-481e-a30a-e6526779a70d.jpg)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
